cmake/Modules/Packages/MLIAP.cmake

```diff
@@ -0,0 +1,29 @@
+# if PYTHON package is included we may also include Python support in MLIAP
+set(MLIAP_ENABLE_PYTHON_DEFAULT OFF)
+if(PKG_PYTHON)
+  find_package(Cythonize)
+  if(Cythonize_FOUND)
+    set(MLIAP_ENABLE_PYTHON_DEFAULT ON)
+  endif()
+endif()
+
+option(MLIAP_ENABLE_PYTHON "Build MLIAP package with Python support" ${MLIAP_ENABLE_PYTHON_DEFAULT})
+
+if(MLIAP_ENABLE_PYTHON)
+  find_package(Cythonize REQUIRED)
+  if(NOT PKG_PYTHON)
+    message(FATAL_ERROR "Must install PYTHON package for MLIAP with Python support")
```

Comment:
The message reads a bit strange.

Reply:
It does the job. I will review all the install instructions at the end and try to harmonize them.

Reply:
Thought: we might change the run-time behavior to put a message in like this right now. As-is if MLIAP+python support is not enabled, it will just give "Invalid Pair Style".

Reply:
I would maybe say "Python package must be enabled for MLIAP with Python support"